src/routes/PostDetail/Comments/Comment/Comment.js

```diff
@@ -76,9 +96,11 @@ export class Comment extends Component {
       { icon: 'Trash', label: 'Delete', onClick: isCreator ? () => deleteComment(comment.id) : null },
       { icon: 'Trash', label: 'Remove', onClick: !isCreator && canModerate ? () => removeComment(comment.id) : null }
     ])
+    const selectedCommentId = findCommentId(window.location.pathname)
```

Comment:
shouldn't this be availing in routeParams? seems slightly off to search through window.location.pathname, especially here in the render (also could be passed in from the connector)

Reply:
I checked `routeParams` and it wasn't there. I am assuming that it needs to be specifically declared in our routes for React Router to pick it up as a param, which involves some more optional matching and complicates routing even further (which perhaps I'm just about to get stuck into with Loren anyway). But ultimately, why feed it through routeParams, or the connector, when you can just get it straight from the source, without using additional libraries etc?

Reply:
yes you would need to add the optional matching. that seems more idiomatic and aligned with how we currently do things and how routing is supposed to work to me... @lorenjohnson curious your thoughts here